data/hai/hai reveal 7 epilogue.txt

```diff
@@ -0,0 +1,820 @@
+				accept
+	npc accompany save
+		personality escort timid
+		government "Free Worlds"
+		ship "Hauler III" "F.S. Terebinth"
+		ship "Hauler III" "F.S. Tamarack"
+		ship "Hauler III" "F.S. Juniper"
+
+
+mission "HRF 12 Escort Free World Outfitters"
+	landing
+	autosave
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
+	source "Rust"
+	destination "Mooncreek"
+	passengers 1
+	on visit
+		dialog `You have arrived without the convoy you are escorting. You must take off and wait for it to continue.`
+	to offer
+		has "HRF 11 Escort Free World Outfitters: done"
+	on offer
+		conversation
+			`On Rust it is tempting to take the time to go and catch up with Barmy Edward but the Kraz convoy were prepared a day early and were near enough to meeting you in orbit.`
+			`	While you are briefly touched down though you are greeted by a message from Alondo, "<first>! Navy intelligence has, in their infinite wisdom, deigned to let us know about a suggested pirate threat. The marauders got outmaneuvered when Mooncreek went up so quickly, it seems they might attempt to intercept the convoy. Please be safe out there!"`
+			`	It seems you'll have to keep your eyes peeled on your way back.`
+				accept
+	on fail
+		dialog "You have failed to safely escort the convoy of Free World outfitters."
+	npc accompany save
+		personality escort timid
+		government "Free Worlds"
+		ship "Hauler III" "F.S. Terebinth"
+		ship "Hauler III" "F.S. Tamarack"
+		ship "Hauler III" "F.S. Juniper"
+		ship "Hauler III" "F.S. Ponderosa"
+		ship "Hauler III" "F.S. Bartlett"
+		ship "Hauler III" "F.S. Sycamore"
+	npc 
+		government "Pirate"
+		system Shaula
+		personality heroic unconstrained waiting nemesis
+		fleet "Marauder fleet X"
+		fleet "Marauder fleet IX" 2
+	npc 
+		government "Pirate"
+		system Gienah
+		personality heroic unconstrained waiting nemesis
+		fleet "Marauder fleet X"
+		fleet "Marauder fleet IX" 2
+	npc 
+		government "Pirate"
+		system Alcyone
+		personality heroic unconstrained waiting nemesis
+		fleet "Marauder fleet X"
+		fleet "Marauder fleet IX" 2
+	npc 
+		government "Pirate"
+		system Almaaz
+		personality heroic unconstrained waiting nemesis
+		fleet "Marauder fleet X"
+		fleet "Marauder fleet IX" 2
+
+
+event "mooncreek main outfitters"
+	planet Mooncreek
+		add outfitter "Lovelace Security"
+		add outfitter "Syndicate Security"
+		add outfitter "Hai Security"
+		security 0.8
+
+#Kraz & Delta V
+event "mooncreek fw outfitters"
+	planet Mooncreek
+		add outfitter "FW Security"
+
+
+#add syndicate hai and republic outfitters event - do not add FW outfitter event - proceed to trigger story conclusion.
```

Comment:
Is this done? I was originally thinking "oh, this is meant to be added when the Syndicate/Republic campaigns are completed," but the inclusion of Hai made me second guess - there's no Hai campaign to be completed besides this one, no?

Reply:
I'm going to make an Unfettered campaign ^^ and in part3 stuff will prob happen with Hai